packages/starrating/src/react/half-star-light.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+
+const HalfLightStar = () => (
```

Comment:
Could we remove the 2nd star if we moved styling into css?  That would allow us to color one shape and only need that one shape in svg.